### DISCUSSION TITLE
How to set jitsi-meet's "desktopSharingFrameRate" min and max values in Pàdé

### DISCUSSION THREAD

**Discussion by MagicD3VIL** [igniterealtime/openfire-pade-plugin#274] [Q&A] [answered]
Hello,
I found in the jitsi-meet's `config.js` option called "desktopSharingFrameRate" with min and max options but after setting it directly in that config and restarting the server the p2p call does not want to start anymore so I had to comment it back. Is it possible to use this setting in the Openfire System Properties? For example `org.jitsi.videobridge.ofmeet.desktopsharingframerate.min = 5`? Or is there another way or location where I can set min and max framerate so the call is not stuck on 5 fps?

Thanks in advance

**Comment by gjaekel** [comment, marked as answer]
You can't configure it at the moment via UI, but we may add it. Please rise an Issue after you try the feature successful.

Meanwhile you may use a developer instrument; you may add it to the file `.../plugins/pade/jitsi-meet/config_custom.js` as a line like
```
config.desktopSharingFrameRate = { min: 5, max: 25 };
```


Be aware that changes to this file get lost every time you restart the Plugin via the Plugin Manager, because this will remove and re-rollout the downloaded archive. Restart whole OpenFire instead if a restart of the Plugin is needed.

**Reply by MagicD3VIL** [reply]
Thanks! Worked perfectly.

**Reply by gjaekel** [reply]
Thank you for feedback, filed as #275 .

**Reply by gjaekel** [reply]
Implemented and prepared for the next release by @deleolajide
